modules/social_features/social_profile/config/install/social_profile.settings.yml

```diff
@@ -1,2 +1,5 @@
 social_profile_show_email: true
 social_profile_show_language: true
+enable_profile_tagging: true
```

Comment:
@dmytriikaiun To keep backward compatibility I think we need to set FALSE by default for all those settings.